src/TestProblem/Hydro/Riemann/Init_TestProb_Hydro_Riemann.cpp

```diff
@@ -472,6 +475,12 @@ void SetGridIC( real fluid[], const double x, const double y, const double z, co
    fluid[ MomIdx[1] ] = Prim[ MomIdx[1] ]*fluid[DENS];
    fluid[ MomIdx[2] ] = Prim[ MomIdx[2] ]*fluid[DENS];
    Pres               = Prim[ ENGY      ];
+#  ifdef YE
+   fluid[ YE        ] = Prim[ YE        ]*fluid[DENS];
+#  endif
+#  ifdef TEMP_IG
+   fluid[ TEMP_IG   ] = Prim[ TEMP_IG   ];
```

Comment:
- [ ] Remove `fluid[TEMP_IG] = NULL_REAL;` at line 492
- [ ] Add a comment explaining the reason for setting `fluid[TEMP_IG]` here first and then overwriting it at line 493.

Reply:
- Removed `fluid[TEMP_IG] = NULL_REAL`.
- I added comments at that lines.